c7n/resources/__init__.py

```diff
@@ -93,6 +93,10 @@ def load_providers(provider_types):
         from c7n_tencentcloud.entry import initialize_tencentcloud
         initialize_tencentcloud()
 
+    if should_load_provider('terraform', provider_types, no_wild=True):
+        from c7n_left.entry import initialize_iaac
+        initialize_iaac()
```

Comment:
nit: The few uses of iaac here seem halfway between iac (infrastructure as code) and iaas (infrastructure as a service). Good chance there's another meaning I'm not considering though.

Reply:
yeah.. I've been remiss in using this typo (iaac) constantly.